hbase/Dockerfile

```diff
@@ -103,7 +103,8 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # krb5-workstation for Kerberos support
 RUN microdnf update && \
     microdnf install \
-    krb5-workstation && \
+    krb5-workstation \
+    tar && \
```

Comment:
Please add a comment explaining what tar is needed for

Reply:
Done.